styles.css

```diff
@@ -93,7 +98,253 @@ ul.navbar-items {
   line-height: 1.5rem;
 }
 
-@media only screen and (max-width: 375px) {
+/* Works section styling */
+
+#works-section {
+  margin: 10px 0;
+}
+
+.box-container {
+  display: flex;
+  margin: 50px 200px 50px 200px;
+}
```

Comment:
- [ ] Great work styling your works container 💯. However, please kindly ensure to use the same `font-family` and `font-color` suggested in the Figma design. 🤓

Figma design 👇🏽
![f1](https://github.com/Xghayor/Portfolio/assets/116701485/99934de9-6d4b-4dc0-a9e1-aaacd48614d6)

Your design 👇🏽
![d1](https://github.com/Xghayor/Portfolio/assets/116701485/3dbfc6bf-fbae-49b6-98a2-62c8e7e6800b)